package.json

```diff
@@ -12,9 +12,6 @@
     "lint": "eslint src/",
     "es-check": "es-check es5 dist/*.js"
   },
-  "publishConfig": {
-    "registry": "https://npm.pkg.github.com/"
-  },
```

Comment:
I'm assuming removing this and using the node action config to set it means the package won't have a reference to a registry anywhere in it? Hence why you can just build once.

Reply:
It breaks publishing locally since the registry is unknown, but fixes the hard-coded registry when attempting both NPM/GPR in actions.